modules/renter/contractor/allowance.go

```diff
@@ -65,7 +65,7 @@ func (c *Contractor) SetAllowance(a modules.Allowance) error {
 	// set the current period to the blockheight if the existing allowance is
 	// empty
 	if reflect.DeepEqual(c.allowance, modules.Allowance{}) {
-		c.currentPeriod = c.blockHeight
+		c.currentPeriod = c.blockHeight - a.RenewWindow
```

Comment:
Currently the siatest is failing due to an underflow where the test package blockheight is less than the allowance renew window.  Should I bother checking for underflow here or can we assume that will never happen in real life because that would mean someone is setting their renew window to something like 3 years.